examples/maather_work_area/genzsprtwo-34.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
This might be suitable for simple cases, but we can make use of shutil module and glob to handle file operations and to search for files with specific extensions.

Reply:
I added shutil so that I could utilize shutil.move instead of renaming files. While glob could be used for filtering files, in this case, relying solely on os functions is sufficient for the task, and it also makes the code easier to read.

Reply:
I added shutil so that I could utilize shutil.move instead of renaming files. While glob could be used for filtering files, in this case, relying solely on os functions is sufficient for the task, and it also makes the code easier to read.